includes/wf_crm_admin_help.inc

```diff
@@ -425,6 +425,14 @@ class wf_crm_admin_help {
       '</p>';
   }
 
+  public static function multivalue_fieldset_create_mode() {
+      print '<p>' .
+        t('Create/ Edit Mode (Default): Pre-populate the existing entry (if any) in order to allow user modifying it. If there isn\'t any existing entry, any value entered into the fieldset will be created as a new entry.') .
```

Comment:
The convention with `t()` is that you don\'t escape quotes when possible. Putting this string in double quotes would avoid the need to escape the single quote.